codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/CodegenEngine.java

```diff
@@ -117,6 +120,20 @@ public void run() {
                         .formatted(lang.name()));
             }
         }
+
+        if (propertiesFile.isPresent()) {
+            final String propertiesTemplate = IoUtils.readUtf8Resource(
+                    this.getClass(), "/templates/project.properties.template");
```

Comment:
Is there a way to have this file not stomp on an existing file?
Or is that just "don't pass the value"?

Reply:
I was already planning to error if the file already exists :)

Reply:
Ended up not doing that because all models build the StandardLibrary transitively, so it gets built many times over in CI. :( Since code generation also stomps on existing files I think we should address this in a separate PR.